webapp/src/views/EventRegistration.vue

```diff
@@ -85,6 +95,8 @@ export default defineComponent({
     const familyName = ref("");
     const programId = ref(0);
     const error = ref("");
+    const warning = ref("");
+    const registerButtonText = ref("Anmelden");
```

Comment:
Making variables like these a `ref` is error prone because with later modifications, developers tend to forget to update them in some places.

Is it possible to turn `alreadyRegistered` into a single `computed` property and handle button text and warning in the HTML template section?